sdk/python/tests/integration/offline_store/test_offline_push.py

```diff
@@ -0,0 +1,226 @@
+import random
```

Comment:
slight preference for naming this file `test_offline_write.py` as this behavior is independent of the push API